platforms/enterprise/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/GradleEnterprisePluginCheckInFixture.groovy

```diff
@@ -16,210 +16,24 @@
 
 package org.gradle.internal.enterprise
 
-import org.gradle.api.Plugin
-import org.gradle.api.initialization.Settings
-import org.gradle.execution.RunRootBuildWorkBuildOperationType
+
```

Comment:
💅 Nit: Extra newline

Reply:
Fixed in https://github.com/gradle/gradle/pull/27365/commits/c47fc966a53fcd3fc7696cecc943d8523f97126c